server/src/metadata/tenant-initialisation/tenant-initialisation.service.ts

```diff
@@ -90,4 +97,38 @@ export class TenantInitialisationService {
       ),
     );
   }
+
+  private async prefillWorkspaceWithStandardObjects(
+    dataSourceMetadata: DataSourceMetadata,
+    workspaceId: string,
+  ) {
+    const objects =
+      await this.objectMetadataService.getObjectMetadataFromDataSourceId(
+        dataSourceMetadata.id,
+      );
+
+    const worksapceDataSource =
+      await this.dataSourceService.connectToWorkspaceDataSource(workspaceId);
+
+    objects.forEach((object) => {
```

Comment:
@Weiko I'm also using `forEach` but as you suggest above it can lead to some inconsistency, we should consider only using `for...of` loop that is more modern javascript